test/python/compiler/test_assembler.py

```diff
@@ -345,12 +326,22 @@ def _qobj_inst_params(expt_no, inst_no):
         self.assertEqual(_qobj_inst_params(7, 0), [1, 0])
         self.assertEqual(_qobj_inst_params(8, 0), [2, 1])
 
+    def test_init_qubits(self):
+        """Check that the ``init_qubits`` assemble option is passed on to the qobj."""
```

Comment:
These docstrings are printed on the screen, not for generate documentation:
```suggestion
        """Check that the init_qubits assemble option is passed on to the qobj."""
```